renovate.json

```
@@ -25,6 +25,10 @@
     {
       "matchPackageNames": ["google.golang.org/genproto/googleapis/**"],
       "groupName": "googleapis"
+    },
+    {
+      "matchPackageNames": ["go.opentelemetry.io/otel/**"],
+      "allowedVersions": "/^v[0-9]+\\.[0-9]+\\.[0-9]+$/"
```

Shouldn't we also allow e.g. `-rc.1` version?
```suggestion
      "allowedVersions": "/^v[0-9]+\\.[0-9]+\\.[0-9]+/"
```